examples/getting-started-movielens/inference-HugeCTR/README.md

```diff
@@ -1,6 +1,6 @@
-# Inference with HugeCTR Model
+# Training and Inference with HugeCTR Model
 
-In this folder, we provide two example notebooks, [movielens-HugeCTR](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-HugeCR.ipynb) and [movielens-HugeCTR-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-inference.ipynb), and explain the steps to do inference with Merlin Inference API after training a model with HugeCTR framework. 
+In this folder, we provide two example notebooks, [Training-with-HugeCTR](https://github.com/rnyak/NVTabular/blob/fix-hugectr-nbs/examples/getting-started-movielens/inference-HugeCTR/Training-with-HugeCTR.ipynb) and [Triton-Inference-with-HugeCTR](https://github.com/rnyak/NVTabular/blob/fix-hugectr-nbs/examples/getting-started-movielens/inference-HugeCTR/Triton-Inference-with-HugeCTR.ipynb), and explain the steps to do inference with Merlin Inference API after training a model with HugeCTR framework. 
```

Comment:
We shouldn't have the links going to your fork here -

Reply:
oh definitely, good catch, will fix it asap :)